lib/internal/fs/glob.js

```diff
@@ -133,7 +132,7 @@ class Pattern {
   isLast(isDirectory) {
     return this.indexes.has(this.last) ||
       (this.at(-1) === '' && isDirectory &&
-      this.indexes.has(this.last - 1) && this.at(-2) === lazyMinimatch().GLOBSTAR);
+      this.indexes.has(this.last - 1) && this.at(-2) === lazyMicromatch().GLOBSTAR);
```

Comment:
I don't think micromatch has a `.GLOBSTAR` export.

Reply:
> I don't think micromatch has a `.GLOBSTAR` export.

Yeah, do you think I should I add a definition somewhere to `fs` constants, or in an upstream PR to micromatch? Something like…
~~~ js
const GLOBSTAR = Symbol('globstar **');
~~~

What do you think? @RedYetiDev

Reply:
I'm not sure, I'm not a core collaborator, but <https://github.com/nodejs/node/blob/main/doc/contributing/using-symbols.md> may help.

Reply:
> Yeah, do you think I should I add a definition somewhere to `fs` constants, or in an upstream PR to micromatch? Something like…

Let me know if you want me to help with anything